universum/analyzers/mypy.py

```diff
@@ -0,0 +1,79 @@
+import argparse
+import glob
+import sys
+import subprocess
+
+from universum.analyzers import utils
+
+
+class MypyAnalyzer:
+    """
+    Mypy runner.
+    Specify parameters such as project folders, config file for code report tool.
+    For example:
+    ./mypy.py --python-version 2 --files *.py tests/
+    Output: json of the found issues in the code.
+    """
+
+    @staticmethod
+    def define_arguments():
```

Comment:
The arguments here are almost exactly the same, as module 'pylint' arguments. Have you considered moving them to 'utils'? E.g. to avoid almost identical, yet quite long explanation of what '--python-version' is

Reply:
yes, I did. Also, "--files" argument is common with pylint and uncrustify. However, I consider refactoring of analyzers a separate item from adding mypy. I think they should be done as distinct PRs

Reply:
My priorities for now are:
1. Check that mypy handler correctly adds comments for review (not sure how to reproduce that at the moment)
2. Expand test suite for analyzers within this PR - right now it's very limited
3. Refactor analyzers as a separate PR